RockWeb/Blocks/Communication/CommunicationEntry.ascx.cs

```diff
@@ -1257,6 +1270,26 @@ private Rock.Model.Communication UpdateCommunication(RockContext rockContext)
             return communication;
         }
 
+        private void ValidateFutureDelayDateTime()
+        {
+            DateTime? futureSendDate = dtpFutureSend.SelectedDateTime;
+            TimeSpan? futureTime = dtpFutureSend.SelectedTime;
+
+            if ( futureTime.HasValue && !futureSendDate.HasValue )
+            {
+                cvDelayDateTime.IsValid = false;
+                cvDelayDateTime.ErrorMessage = "The Delay Date must also be selected with time." ;
+                return;
+            }
+
+            if ( futureSendDate.HasValue && futureSendDate.Value.CompareTo( RockDateTime.Now ) < 0 )
+            {
+                cvDelayDateTime.IsValid = false;
+                cvDelayDateTime.ErrorMessage = "The Delay Date Time Selected Could not be of past.";
+                return;
```

Comment:
Can you change this text to "The Delay Send Until value must be a future date/time"